### DISCUSSION TITLE
How do you appropriately align when annotating equations in the following way?

### DISCUSSION THREAD

**Discussion by cyber-rhythms** [KaTeX/KaTeX#3201] [Q&A] [answered]
I would like to annotate my equations to include Roman numbering in brackets over the equals sign, like so:

![katexalign](https://user-images.githubusercontent.com/36315073/131169265-54270857-ce00-4cf0-9106-d9d41d3269b3.png)

This was produced using the following code:

```
$$\begin{aligned}
\mathbb{P}_{i \sim D_1} \left( H(x_i) \neq y_i \right) &= \sum^m_{i=1} D_1(i) \cdot \mathbb{I} \left(H(x_i) \neq y_i \right) \\
&\overset{(i)}{=} \sum^m_{i=1} D_1(i) \cdot \mathbb{I} \left(y_i F(x_i) \leq 0 \right) \\
&\overset{(ii)}{\leq} \sum^m_{i=1} D_1(i) \exp(-y_i F(x_i)) \\
&\overset{(iii)}{=} \sum^m_{i=1} \left( D_{T+1}(i) \prod^T_{t=1} Z_t \right) \\
&= \prod^T_{t=1} Z_t \left( \sum^m_{i=1} D_{T+1}(i) \right)\\
&= \prod^T_{t=1} Z_t. \\
\end{aligned}$$
```

You will note that the use of `\overset{(iii)}{\leq}` etc. causes the alignment to be thrown off. Is there a way around this in KaTeX?

**Comment by edemaine** [comment, marked as answer]
`\overset{\clap{(iii)}}{\leq}` ought to do what you want.

**Reply by cyber-rhythms** [reply]
Thank you @edemaine, after 2+ hours of frustrated experimentation I was on the verge of giving up.

**Reply by cyber-rhythms** [reply]
I tried to mark your reply as the answer, but I am unable to do so as I am experiencing a GitHub glitch similar to [this one](https://github.com/github/feedback/discussions/2903).
